monasca-alarms/definitions.yml

```diff
@@ -0,0 +1,193 @@
+#
+# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+#
+# Default Notifications and Alarms
+#
+notifications:
+  - name: root email
+    type: email
+    address: root@localhost
```

Comment:
My only real concern here, will there be issues if notifications can't be sent? `root@localhost` will never be valid, so I'd like to be  sure it won't cause trouble if the defaults are used.